src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -563,7 +593,7 @@
 	public static final RegistryObject<Item> SCALEMATE_CINNAMONWHIFF = REGISTER.register("scalemate_cinnamonwhiff", () -> new ScalemateItem(new Item.Properties()));
 	public static final RegistryObject<Item> SCALEMATE_HONEYTONGUE = REGISTER.register("scalemate_honeytongue", () -> new ScalemateItem(new Item.Properties()));
 	public static final RegistryObject<Item> SCALEMATE_LEMONSNOUT = REGISTER.register("scalemate_lemonsnout", () -> new ScalemateItem(new Item.Properties()));
-	public static final RegistryObject<Item> SCALEMATE_PINESNOUT = REGISTER.register("scalemate_pinesnout", () -> new ScalemateItem(new Item.Properties()));
+	public static final RegistryObject<Item> SCALEMATE_PINESNORT = REGISTER.register("scalemate_pinesnort", () -> new ScalemateItem(new Item.Properties()));
```

Comment:
Changing its registry name breaks backwards-compatibility. Make sure to either have the old name map to the new name through `MissingMappingsEvent`, or keep the old registry name and add a todo(that is revert the string passed to `register()`. the field name and the name in `MinestuckEnUsLanguageProvider` doesn't have to be changed back).

Reply:
:?? < how and where do i do the MissingMappingsEvent thing? would rather not make the registry change a todo

Reply:
You would make a new method in MSItems. However that might be a bit outside your wheelhouse for the time being. My recommendation is to leave it as is or leave it as is but add a TODO since its not a weapon(the focus of this PR)

Reply:
:33 < ehhhh this has b33n an issue for a long while so id like to rectify it as soon as pawssible. ill revert the registry back to pinesnout fur now but ill k33p the name in the language purrvider and item sprite the same